packages/dtslint/test/dt-header.test.ts

```diff
@@ -28,16 +28,37 @@ ruleTester.run("dt-header", dtHeader, {
       `,
       errors: [
         {
-          column: 2,
+          column: 1,
           data: {
             expected: "/\\/\\/ Type definitions for (non-npm package )?/",
           },
-          line: 1,
+          line: 2,
           messageId: "parseError",
         },
       ],
       filename: "types/only-comment/index.d.ts",
     },
+    {
```

Comment:
As per the above comment, I was wondering if `parsimmon` changed how they parse whitespace (maybe they were now ignoring it?), but it seems they didn't, and this test makes sure of that. So it seems the only change is in the position of the reported error.